.travis.yml

```
@@ -0,0 +1,52 @@
+language: julia
+
+os:
+  - linux
+  - osx
+  - freebsd
+
```

Add also Windows here:
```suggestion
   - windows

```